tools/osv-linter/test_data/CVE-2018-5407.json

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
Most of these test files don't seem to be referenced anywhere as part of an automated test. Are you planning to add them in another PR?

Reply:
There's so much extra scaffolding to add around CI/CD type stuff... Yes, that'll be in a future PR. Right now, these test files are for running the code against manually, per the PR description.